src/github.org

```diff
@@ -0,0 +1,104 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi and Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, and handles merging
```

Comment:
Don't talk about merging yet, it's way too early. Maybe:
```
Git tracks changes made to files in your project, so you can access old versions at any time without fear.
```

At most, I would say something like 'allows multiple developers to work on the same code'